<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editDialog</class>
 <widget class="QDialog" name="editDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit person</string>
  </property>
  <widget class="QLineEdit" name="lineEditInside">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInsideDeath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Death year:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInsideBirth">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Min 1791</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>59</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Birth year:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>36</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="idLabel">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>89</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButtonFemale">
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonMale">
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonOther">
      <property name="text">
       <string>Unknown</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>169</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="savePushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelPushButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_dialogPersonError">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
